src/styles/input.css

```diff
@@ -0,0 +1,41 @@
+.ui.input > input {
+  padding: var(--spacing-1) var(--spacing-4) var(--spacing-1) var(--spacing-2);
+  line-height: 24px;
+  border-radius: var(--border-radius-2);
+  background-color: var(--shade-white);
+  border: 1px solid var(--neutral-900);
+  font-size: 16px;
```

Comment:
Can we make variables for this and then reference them?
```
--font-size: 16px;
--line-height: 24px;
```

Reply:
Definitely!